docs/v3/status-socket.rst

```diff
@@ -0,0 +1,53 @@
+Status Socket
+=============
+
+Basic information about running OnionBalance can be obtained by querying
+so called status socket.
+
+Status socket is a Unix socket file created by OnionBalance. It is
+automatically closed by OnionBalance after reading it to the end.
+
+Example:
+
+.. code-block::
+
+    socat - unix-connect:/var/run/onionbalance/control | json_pp -json_opt pretty
+    {
+       "services" : [
+          {
+             "instances" : [
+                {
+                   "introModified" : "2020-06-16 19:59:28",
+                   "ipsNum" : 3,
+                   "onionAddress" : "vkmiy6biqcyphtx5exswxl5sjus2vn2b6pzir7lz5akudhwbqk5muead.onion"
+                }
+             ],
+             "onionAddress" : "bvy46sg2b5dokczabwv2pabqlrps3lppweyrebhat6gjieo2avojdvad.onion.onion",
+             "publishAttemptFirst" : "2020-06-16 20:00:12",
+             "publishAttemptSecond" : "2020-06-16 20:00:12"
+             "uploadedFirst" : "2020-06-16 20:00:12",
```

Comment:
I think `uploadedFirst` and `uploadedSecond` are not meaningful here and they are exactly the same as `publishAttemptFirst` and `publishAttemptSecond`. Perhaps we can remove them for now.

Reply:
From https://github.com/DonnchaC/onionbalance/issues/60:

> 2. the time of the last successful non-zero-sized master descriptor upload, or a explanation if none
> 3. the time of the last attempt to assemble a new master descriptor, and status thereof?

These are considered two distinct things here. You say they are the same.

Originally there was only `uploaded*`. `publishAttempt*` was added later by me. The distinction is between _successful_ upload and _attempt_ to upload.

So, remove it or leave as is? Please make sure because I will squash commit and it would be hard to revert the change!

Reply:
Please remove the `uploadedFirst` and `uploadedSecond`. Right now in Onionbalance there is no way to learn if the upload actually succeeded. So let's only announce the attempts and not pretend they are successes.